src/metabase/lib/drill_thru/column_filter.cljc

```diff
@@ -36,6 +34,10 @@
   {:type       :drill-thru/column-filter
    :initial-op initial-op})
 
-(defmethod lib.drill-thru.common/drill-thru-method :drill-thru/column-filter
-  [query stage-number {:keys [column] :as _drill-thru} filter-op value & _]
-  (lib.filter/filter query stage-number (lib.options/ensure-uuid [(keyword filter-op) {} (lib.ref/ref column) value])))
```

Comment:
This was basically just duplicating the logic in `lib.filter/filter-clause` so I just updated `filter-clause` to take keywords or strings as well and removed the duplication.